src/openag_brain/software_modules/recipe_handler.py

```diff
@@ -105,6 +108,43 @@ def __iter__(self):
             yield (time.time(), variable, value)
         yield (time.time(), RECIPE_END.name, self.id)
 
+def hrs_to_seconds(hrs):
+    return hrs * (60 * 60)
+
+def days_to_seconds(days):
+    return hrs_to_seconds(days * 24)
+
+@interpret_recipe.register("phased")
+class PhasedRecipeInterpreter:
+    def __init__(self, recipe, start_time=None, timeout=1):
+        init_time = rospy.get_time()
+        start_time = start_time or init_time
+        if start_time > init_time:
+            raise ValueError("Recipes cannot be scheduled for the future")
+        self.start_time = start_time
+        self.stages = recipe["operations"]
+        self.id = recipe["_id"]
+        self.timeout = timeout
+
+    def __iter__(self):
+        yield (rospy.get_time(), RECIPE_START.name, self.id)
+        for stage in self.stages:
+            stage_duration = days_to_seconds(stage["days"])
+            end_of_stage = (rospy.get_time() - self.start_time) + stage_duration
```

Comment:
Upon reflection, I think I'm ok with having `days` become `day_cycles`, and be a multiplier for `day + night`. So `day_cycles: 5` would be valid.

Your argument brings me around: it's hard to reason about solar 24hr days overlayed against artificial day lengths, regardless. Better to reason about phases (you know the plant can handle a 14 hr day in this stage, etc) and how many cycles you want it to go through.